### PR TITLE
Implement Landing Page

### LINKED ISSUE #49: FE: Landing Page
- Create the UI for the landing page (please see mocks)

### PR DESCRIPTION
**Pull request by rj13371**
### What this PR does :
Got the basic layout of the landing page setup. Just need a few suggestions on review before a merge. Had trouble with sizing the image behind the navbar and moving the h1 around but other then that I was able to make a skeleton that I think will be ready to retrieve profiles on our route with a form based on location and schedule.

### Screenshots / Videos:
demo screenshot

### Any information needed to test this feature :
Git clone the repo
cd into client 
Start the client app with npm start 
Now instead of redirecting to login if you have no token, it will redirect you / , which is our landing page. Git clone the repo
cd into client 
Start the client app with npm start 
Now instead of redirecting to login if you have no token, it will redirect you / , which is our landing page. 

### Any issues with the current functionality :
Having trouble styling the dog picture and the header to match the mocks exactly

### CODE REVIEW

File: client/src/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -133,7 +133,9 @@ const Navbar: React.FC = () => {
       container
     >
       <Grid xs={4} md={6} item>
-        <img className={classes.navbarLogo} src={lovingSitterLogo} />
+        <NavLink className={classes.navbarItem} to={'/'}>
+          <img className={classes.navbarLogo} src={lovingSitterLogo} />
```
Comment (rajivtitus) [inline client/src/components/Navbar/Navbar.tsx]: Let's also add an 'alt' attribute to HTML image elements for accessibility reasons

File: client/src/pages/Landing/LandingForm.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+  return (
+    <Grid container>
+      <Grid item xs={6}>
+        <Container>
+          <Typography className={classes.header} variant="h2" component={'h1'}>
+            Find the care your <br />
+            dog deserves
+          </Typography>
+          <Formik
```
Comment (rajivtitus) [inline client/src/pages/Landing/LandingForm.tsx:47]: Good job using Formik here 👍🏽

File: client/src/pages/Landing/LandingForm.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                    {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Find my dog sitter'}
+                  </Button>
+                </Box>
+              </form>
+            )}
+          </Formik>
+        </Container>
+      </Grid>
+      <Grid item xs={6}>
+        <img className={classes.dogImage} src={dogs} />
```
Comment (rajivtitus) [inline client/src/pages/Landing/LandingForm.tsx]: Same as comment above about 'alt' attributes

File: client/src/pages/Landing/LandingPage.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  const handleSubmit = (
+    { location, dropIn, dropOff }: { location: string; dropIn: Date; dropOff: Date },
+    { setSubmitting }: FormikHelpers<{ location: string; dropIn: Date; dropOff: Date }>,
+  ) => {
+    getProfiles(location, dropIn, dropOff).then((data) => {
+      if (data.error) {
+        setSubmitting(false);
+        updateSnackBarMessage(data.error.message);
+      } else if (data.success) {
+        // updateLoginContext(data.success);
```
Comment (rajivtitus) [inline client/src/pages/Landing/LandingPage.tsx]: Please uncomment or remove commented code

**Review by rajivtitus — COMMENTED** [review]
UI looks great on this! Just a couple of minor comments 💯 
Let's also make sure the Navbar matches the mocks 😉

**Comment by sundayezeilo** [comment]
Please resolve merge conflict

**Comment by rj13371** [comment]
I resolved the merge conflict. Should be good to merge into main now?

File: client/src/pages/Landing/LandingPage.tsx [hunk 5]
```diff
@@ -16,7 +15,8 @@ export default function Login(): JSX.Element {
         setSubmitting(false);
         updateSnackBarMessage(data.error.message);
       } else if (data.success) {
-        // updateLoginContext(data.success);
+        // we will handle submit here during integration of profile search
+        console.log('submitted');
```
Comment (sundayezeilo) [inline client/src/pages/Landing/LandingPage.tsx]: Let's get rid of unnecessary console logs.

**Review by sundayezeilo — COMMENTED** [review]
Good work with this one. Thank you for making the requested changes :+1:

File: client/src/components/Navbar/Navbar.tsx [hunk 6]
```diff
@@ -87,7 +86,10 @@ const MenuItem: React.FC<{
 
   return (
     <Grid key={resource} sx={{ textAlign: 'center' }} xs={2} justifySelf="flex-end" item>
-      <NavLink className={classes.navbarItem} to={resource}>
+      <NavLink
+        className={clsx(classes.navbarItem, location.pathname === '/' && classes.navbarItemLanding)}
```
Comment (rajivtitus) [inline client/src/components/Navbar/Navbar.tsx:90]: 👍🏽

File: client/src/components/Navbar/useStyles.ts [hunk 7]
```diff
@@ -8,9 +8,23 @@ export const useStyles = makeStyles((theme: Theme) => ({
     background: 'white',
   },
   transparentNavbar: {
+    position: 'fixed',
+    zindex: 10000,
```
Comment (rajivtitus) [inline client/src/components/Navbar/useStyles.ts]:
```suggestion
    zIndex: 10000,
```
I think this is a typo here

**Review by rajivtitus — APPROVED** [review]
Good work making the requested changes and also updating the PR template with the new UI. A couple of minor comments to address prior to merging 👍🏽

**Pull request merged**